### PR TITLE
add tagging

### PR DESCRIPTION
**Pull request by takose**

### CODE REVIEW

File: lib/hitorigoto_reporter/formatter.rb [hunk 1]
```diff
@@ -17,6 +17,11 @@ def to_markdown
         .join("\n\n\n")
     end
 
+    def stamps
+      filtered_list, file_comments = filter_list_and_extract_comments
+      filtered_list.map{ |h| h.stamps }.flatten.uniq
```
Comment (izumin5210) [inline lib/hitorigoto_reporter/formatter.rb]: I think `Enumerable#flat_map` and [`&` operator](https://www.reddit.com/r/ruby/comments/2ipgx6/help_understanding_mapjoin/) make your code more smart :yum:

**Review by izumin5210 — CHANGES_REQUESTED** [review]

**Comment by takose** [comment]
I have no confidence in this change.. Is this right?

**Comment by takose** [comment]
I retried to fix!

**Review by izumin5210 — APPROVED** [review]

**Comment by izumin5210** [comment]
Good job !
寿司ゆき:ok

**Pull request merged**